software/tegon.yml

```
@@ -0,0 +1,11 @@
+name: "Tegon"
+website_url: "https://tegon.ai"
+source_code_url: "https://github.com/tegonhq/tegon"
+description: "Tegon is an open-source, dev-first alternative to Jira and Linear."
```

Trying to get rid of the term 'open-source' (https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/CONTRIBUTING.md#other-guidelines)
```suggestion
description: "Dev-first issue tracking tool (alternative to Jira, Linear)."
```